instrumentation/undertow/javaagent/undertow-javaagent.gradle

```diff
@@ -4,7 +4,8 @@ muzzle {
   pass {
     group = "io.undertow"
     module = 'undertow-core'
-    versions = "[2.0.0.Final,)"
+    versions = "[1.4.0.Final,)"
```

Comment:
Change README as well, please

Reply:
added undertow to supported-libraries.md